globals/card_definitions.gd

```diff
@@ -22,6 +22,8 @@ func get_random_deck(season : int):
 	else:
 		var season_decks = []
 		for deck in decks:
+			if deck['id'] in GlobalSettings.CharacterBanlist:
```

Comment:
You need to do this for season == -1 above too.
Maybe just a while loop until you get a deck that isn't banned.